portfolio/src/main/webapp/testimonials.html

```diff
@@ -24,9 +23,13 @@
     <script src="https://aframe.io/releases/1.0.4/aframe.min.js"></script>
     <script src="https://unpkg.com/aframe-environment-component@1.1.0/dist/aframe-environment-component.min.js"></script>
 
+    <!-- Voice Control -->
+    <script src="//cdnjs.cloudflare.com/ajax/libs/annyang/2.6.0/annyang.min.js"></script>
```

Comment:
Is `voiceControl.js` ever being used?

Reply:
No- see comment above. I was playing around with[ this file ](https://github.com/googlearchive/webplatform-samples/tree/master/webspeechdemo)that I found on github but decided annyang was the better way to go.